cl8-web/server/src/config.js

```diff
@@ -1,4 +1,5 @@
-const admin = require('firebase-admin')
+const admin = require('firebase-admin');
```

Comment:
Looks like prettier is getting ignored. I have no preference for style (well, I do, but it's not important), just trying to keep the commit noise down.